en/01_Optuna_Quick_Start.ipynb

```diff
@@ -0,0 +1,358 @@
+    "\n",
+    "Note that `objective` is a blackbox function for Optuna. The library only observes the input, `x`, and the output of the function.  The library gradually improves `x` with a smart internal algorithm (Bayesian optimization).\n",
+    "\n",
+    "You can access to the best result with `study.best_value` and `study.best_params`, which should be near $x = 2$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('Minimum objective value: ' + str(study.best_value))\n",
+    "print('Best parameter: ' + str(study.best_params))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In summary, you need the following steps to set up the optimization.\n",
+    "\n",
+    "- Define the objective function that calculate the minimization/maximization target.\n",
+    "- Inside the objective function, get the hyperparameters with `suggest` methods.\n",
+    "- Instantiate the `study` object.\n",
+    "- Start the optimization with `study.optimize`, specifying the number of trials with `n_trials`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "2zuF5-nnvqBu"
+   },
+   "source": [
+    "## Optimize Machine Learning Models\n",
+    "\n",
+    "Let's optimize the following machine learning logic, where a linear regression model (Lasso) is trained for the Boston Housing dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "Spk8G3Y9rxRv"
+   },
+   "outputs": [],
+   "source": [
+    "import sklearn.datasets\n",
+    "import sklearn.linear_model\n",
+    "import sklearn.metrics\n",
+    "\n",
+    "# hyperparameter setting\n",
+    "alpha = 1.0\n",
+    "\n",
+    "# data loading and train-test split\n",
+    "X, y = sklearn.datasets.load_boston(return_X_y=True)\n",
+    "X_train, X_val, y_train, y_val = sklearn.model_selection.train_test_split(X, y, random_state=0)\n",
+    "\n",
+    "# model training and evaluation\n",
+    "model = sklearn.linear_model.Lasso(alpha=alpha)\n",
+    "model.fit(X_train, y_train)\n",
+    "y_pred = model.predict(X_val)\n",
+    "error = sklearn.metrics.mean_squared_error(y_val, y_pred)\n",
+    "\n",
+    "# output: evaluation score\n",
+    "print('Mean squared error: ' + str(error))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Performance of Lasso regression is sensitive to the L1 constant, `alpha`, and it's tiresome for humans to manually search this hyperparameter. With Optuna, you can search the appropriate value of `alpha` as follows. Note that you just need to wrap the machine learning logic in the previous cell with `objective` and to set up `study` object."
```

Comment:
I'm not sure but maybe `find` or `search for` is more suitable for the sentence `you can search the appropriate value` than just `search`. In my understanding, this sentence means `you can search (the hyperparameter space) for the appropriate value`.